docs/api/status/api-reference.md

```diff
@@ -0,0 +1,19 @@
+# Status service API reference
+
+By default the `/status` endpoint will simply return a HTTP status code of 200 without a response text, acting as a health endpoint for the HTTP API. 
```

Comment:
this is no longer true, it returns an empty json object

Reply:
fixed